### PR TITLE
feature: add annotation blocker plugin to bcs-webhook-server; issue #…

### PR DESCRIPTION
**Pull request by abstractmj**
…1164

### CODE REVIEW

File: bcs-runtime/bcs-k8s/bcs-component/bcs-webhook-server/internal/plugin/blockannotation/expr.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
+package blockannotation
```
Comment (silenceper) [inline bcs-runtime/bcs-k8s/bcs-component/bcs-webhook-server/internal/plugin/blockannotation/expr.go:1]: 同上，缺失License信息

**Review by silenceper — CHANGES_REQUESTED** [review]

File: bcs-runtime/bcs-k8s/bcs-component/bcs-webhook-server/internal/plugin/blockannotation/conf.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package blockannotation
```
[the remark this answers is not in the document]
Comment (abstractmj) [reply bcs-runtime/bcs-k8s/bcs-component/bcs-webhook-server/internal/plugin/blockannotation/conf.go:13]: fixed

File: bcs-runtime/bcs-k8s/bcs-component/bcs-webhook-server/internal/plugin/blockannotation/expr.go [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	if err := json.Unmarshal([]byte(op2), &obj2); err != nil {
+		return false, err
+	}
+	for k, v := range obj1 {
+		tmpV, ok := obj2[k]
+		if !ok {
+			return false, nil
+		}
+		if !reflect.DeepEqual(tmpV, v) {
+			fmt.Printf("%v, %v", tmpV, v)
```
[the remark this answers is not in the document]
Comment (abstractmj) [reply bcs-runtime/bcs-k8s/bcs-component/bcs-webhook-server/internal/plugin/blockannotation/expr.go]: fixed

File: bcs-runtime/bcs-k8s/bcs-component/bcs-webhook-server/internal/plugin/blockannotation/expr.go (hunk 1 above)
Comment (abstractmj) [reply bcs-runtime/bcs-k8s/bcs-component/bcs-webhook-server/internal/plugin/blockannotation/expr.go:13]: fixed

**Review by wenxinlee2015 — APPROVED** [review]

**Review by wenxinlee2015 — APPROVED** [review]

**Review by wenxinlee2015 — APPROVED** [review]

**Review by wenxinlee2015 — APPROVED** [review]

**Review by silenceper — APPROVED** [review]

**Pull request merged**